mandatory/3-function-output.js

```diff
@@ -1,16 +1,15 @@
-// Add comments to explain what this function does. You're meant to use Google!
+//  returns a random number between 0 , and 10 :
```

Comment:
while this statment is partially true, 10 is not included. Math.random() always returns a vlaue between 0 (inclusive) and 1 (exclusive)